frontend/src/containers/ExplorePage/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
I think it would be better to get rid of unused imports to get rid of those warnings